### ISSUE TITLE
Error: Node is either not visible or not an HTMLElement

### ISSUE THREAD

**Issue by focamonca** [social-manager-tools/instagram-bot.js#82]
### Versions

Replace the values below with your own:

- **Bot Version:** v0.9.0
- **Node Version:** lastest
- **Operating System:** Raspbian 9
- **Browser:** chromium

### Expected Behavior

Please describe the program's expected behavior.

```
The bot should run
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
pi@raspberrypi:~/Desktop/instabotjs $ npm update
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"arm"})

+ instagrambotlib@0.9.0-b
updated 1 package and audited 3025 packages in 55.453s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
pi@raspberrypi:~/Desktop/instabotjs $ node bot.js
[INFO] utils: loginpin.txt exist
[INFO] utils: logs/debug.log exist
[INFO] utils: logs/errors.log exist
[INFO] utils: databases/fdf.db exist
[INFO] utils: databases/logs.db exist
[WARNING] utils: Bot work? Please donate for support this project!
[WARNING] utils: Donate with patreon: http://patreon.ptkdev.io
[WARNING] utils: Donate with paypal: http://paypal.ptkdev.io
[INFO] login: loading...
[INFO] login: open_loginpage
[INFO] utils: Bot is updated! :D
[INFO] login: set_username
[INFO] login: set_password
[INFO] login: submit
(node:6988) UnhandledPromiseRejectionWarning: Error: Node is either not visible or not an HTMLElement
    at ElementHandle._clickablePoint (/home/pi/Desktop/instabotjs/node_modules/puppeteer/lib/ElementHandle.js:91:13)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
(node:6988) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:6988) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. cd Desktop/instabotjs
2. node bot.js

### config.js (remember to remove password)
```
module.exports = {
    // InstagramBot.js Configs
    "debug": true,
    "login": true,
    "ui": false, // only for social-manager-tools

    // Instagram Account
    "instagram_username": "", // without @
    "instagram_password": "",
    "instagram_hashtag": ["instagram", "muraleshunter", "hashtag"], //without #, raccomanded more than 20
    "instagram_pin": "sms", //method to receive pin (email or sms)

    // BOT Configs
    // [WORK] likemode_classic          - select random hashtag from config list and like 1 random photo (of last 20) | 400-600 like/day.
    // [WORK] likemode_realistic        - select random hashtag from config list, like fast 10-12 photo and sleep 15-20min. Sleep at night
    // [WORK] likemode_superlike        - select random hashtag from config list and like 3 random photo of same user | 400-600 like/day.
    // [WORK] likemode_competitor_users - select account, select random followers, like 10-12 photo and sleep 15-20min.
    // [WORK] comment_mode              - leave a comment under the photo via hashtag.
    // [WORK] fdfmode_classic           - follow 30 users, and defollow the first followed at 31 follow (in loop). This method is not detected from socialblade. ~1h | 300/300 follow-defollow/day.
    "bot_mode": "likemode_realistic",
    "bot_likeday_min": 800,      // work only in likemode_classic
    "bot_likeday_max": 900,      // work only in likemode_classic. In 2018 limit instagram is 1000/day
    "bot_sleep_night": false,    // if you need stop bot in night 
    "bot_start_sleep": "7:00",   // sleep from 00:00 to 7:00 am, work in likemode_realistic, comment_mode, likemode_superlike and likemode_competitor_users
    "bot_superlike_n": 3,        // superlike number of likes for user
    "bot_followday": 300,        // number of follow-defollow user/day, work in fdfmode_classic. In 2018 limit instagram is 500/day
    "bot_followrotate": 30,      // follow 30 users, and defollow the first followed at 31 follow (in loop), work in fdfmode_classic | If is 0 defollow all followed previously by bot
    "bot_userwhitelist": ["giulyegiammy", "powerslidelover", "ste_tami"],   // usernames without @ | don't defollow this users on follow-defollow mode, work in fdfmode_classic
    "comment_mode": {
        "comments": {
            "type": "array",
            "source": ["comment1", "comment2", "comment3"],
        }
    },
    "likemode_competitor_users": {
        "account": "user_nickname" // without @
    },

    // Puppeteer Configs
    "chrome_headless": true,
    "chrome_options": ["--disable-gpu", "--no-sandbox", "--window-size=1920x1080"],
    "executable_path": "/usr/bin/chromium-browser", // example for Mac OS: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome

    // DATABASES
    "fdfdatabase_path": "./databases/fdf.db", // databases with username of followed users with follodefollow mode
    "logdb_path": "./databases/logs.db",      // logs in sqllite database for better report

    // LOG
    "pin_path": "./loginpin.txt",
    "log_path": "./logs/debug.log",
    "logerr_path":"./logs/errors.log",
    "screenshot_path": "./logs/screenshot/",
    "log": {
        "drivers": ["console"], // slack or console
        "screenshot": false,  // disable or enable screenshot in logs folder
        "channels": {
            "console": "",
            "slack": {
                "webhook": ""
            }
        }
    }
};


```

### Screenshots (Optional)

If the error is graphical in nature it is helpful to provide a screenshot. 
![image](https://user-images.githubusercontent.com/19846692/45442228-c1226400-b6c1-11e8-8170-8885705c2cd8.png)

Sorry for the bother man

**Comment by ptkdev** [comment]
Try downgrade to **puppeteer v1.4.0**
Run `npm install puppeteer@1.4.0` or change version in `package.json` and run `npm update -f`

**Comment by focamonca** [comment]
It worked!

**Comment by ptkdev** [comment]
Nice! This is bug, i fix this in next release **v0.9.1** for remove workaround :)

**Comment by focamonca** [comment]
but I had to replace manually the files from the v1.4.0 downloaded from github

**Comment by NotLuke92** [comment]
@focamonca which files do you replace? I have the same issue on my raspberry but I can't understand if you did what @ptkdev said or what else.

**Comment by ptkdev** [comment]
@NotLuke92 
## METHOD 1:
1. Run `npm install puppeteer@1.4.0`
2. Go to `node_modules/instagrambotlib/node_modules` and delete `puppeeter`
3. Run bot

## METHOD 2:
1. Run `npm install puppeteer@1.4.0`
2. Go to `node_modules` and copy `puppeteer` folder
3. Go 1 folder back and paste&replace into `node_modules/instagrambotlib/node_modules`
3. Run bot

Try this 2 method.

**Comment by ptkdev** [comment]
User removed photo, `https://www.instagram.com/p/BnqMV8lloBU/` don't exist. I know of this bug, i fix this in next release.

**Comment by ptkdev** [comment]
Fixed with **`v0.9.1`** release

<a href="http://patreon.ptkdev.io" alt="Patreon Backer"><img src="https://img.shields.io/badge/donate-patreon-F87668.svg"></a> <a href="http://paypal.ptkdev.io" alt="Paypale Donate"><img src="https://img.shields.io/badge/donate-paypal-46AFE0.svg"></a>  <a href="http://coffee.ptkdev.io" alt="Ko-fi Donate"><img src="https://img.shields.io/badge/buy%20me-coffee-4B788C.svg"></a>
